src/main/java/com/uid2/admin/vertx/service/OperatorKeyService.java

```diff
@@ -184,6 +187,18 @@ private void handleOperatorAdd(RoutingContext rc) {
                 return;
             }
 
+            Set<Role> roles = (!rc.queryParams().contains("roles")
+                    || RequestUtil.getRoles(rc.queryParam("roles").get(0)) == null)
+                        ? new HashSet<>()
+                        : RequestUtil.getRoles(rc.queryParam("roles").get(0));
+
+            if (rc.queryParams().contains("roles")
+                    && rc.queryParam("roles").get(0) != ""
```

Comment:
You can use `!rc.queryParam("roles").get(0).trim().isEmpty()`

`.isBlank()` works as well, but it's a Java 11 feature